wapitiCore/net/page.py

```diff
@@ -211,13 +211,13 @@ def json(self):
 
         try:
             return self._response.json()
-        except ValueError:
-            pass
+        except ValueError as e:
+            logging.exception(e)
```

Comment:
I'm not sure we should log the exception as we have no control on the server responses.

If a server goes mad (5XX errors like internal or gateway errors) during scan I'm afraid it will generate way too much exceptions.